Core/include/Acts/EventData/MultiTrajectoryHelpers.hpp

```diff
@@ -0,0 +1,107 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+#include <functional>
+#include <unordered_map>
+
+#include "Acts/EventData/MultiTrajectory.hpp"
+#include "Acts/Geometry/Layer.hpp"
+#include "Acts/Geometry/TrackingVolume.hpp"
+#include "Acts/Surfaces/Surface.hpp"
+
+namespace Acts {
+
+namespace MultiTrajectoryHelpers {
+
+/// @brief Struct for brief trajectory summary info
+/// @TODO: add nSharedHits
+///
+struct TrajectoryState {
+  size_t nStates = 0;
+  size_t nMeasurements = 0;
+  size_t nOutliers = 0;
+  size_t nHoles = 0;
+  double chi2Sum = 0;
+};
+
+using TrajectoryStateContainer =
+    std::unordered_map<GeometryID::Value, TrajectoryState>;
```

Comment:
Is there a specific reason for using `GeometryID::Value` here? You should be able to use `GeometryID` directly.

Reply:
It's because we only need the volume identifier, right? In principle, we could use `GeometryID` with only volume identifier (no info of layer and sensitive), but that might not be so straightforward for the user.

Reply:
Ah, Ok. That was not clear to me from; neither from the type name or the key type. I would suggest to either add documentation to the `TrajectoryStateContainer`as-is or rename it to a more descriptive name. Something like `VolumeTrajectoryStateContainer` or along similar lines. Just to avoid any similar confusion in the future.

Reply:
Thank you! Done.